UIKit-HW/UIKit-HW/StopwatchViewController/ViewController/LapsTableViewCell/LapsTableViewCell.swift

```diff
@@ -0,0 +1,33 @@
+//
+//  LapsTableViewCell.swift
+//  UIKit-HW
+//
+//  Created by Матвей Белков on 08.07.2021.
+//
+
+import UIKit
+
+class LapsTableViewCell: UITableViewCell {
+
+    //MARK: - Private Properties
+    
+    @IBOutlet weak var timeLabel: UILabel!
+    @IBOutlet weak var lapNumberLabel: UILabel!
+    
+    //MARK: UITableViewCell(LapsTableViewCell)
+
+    override func awakeFromNib() {
+        super.awakeFromNib()
+    }
+
+    override func setSelected(_ selected: Bool, animated: Bool) {
+        super.setSelected(selected, animated: animated)
+    }
+
+    //MARK: - Public Methods
+    
+    func setup(lap: Lap) {
```

Comment:
```suggestion
  private  func setup(lap: Lap) {
```
setup - чего ? view?